Application/src/Parser.java

```diff
@@ -0,0 +1,99 @@
+import Model.diagramObject;
+import com.google.gson.Gson;
+
+/**
+ * @author Rashad Kamsheh & Isabelle Törnqvist
+ * @since 2017-10-16
+ */
+
+public class Parser {
+
+    // odd number for the First Diagram counter
+    static int counter = 1;
+    // even number for the parallel Diagram counter
+    static int parallelCounter = 2;
+
+    static String Processes = "";
+    static String properProcesses = "";
+
+    static String FirstDiagram = "";
+    static String properFirstDiagram = "";
+
+    static String ParallelDiagram= "";
+    static String ProperParallelDiagram = "";
+
+
+    /**
+     * Parses the imported JSON files and prints it to the console
```

Comment:
Printing to the console should not be in the finished version

Reply:
The reasoning is: The Application has a higher failure chance if it interacts with the System.out stream. 
I discovered this earlier today with Sebastian and will probably investigate it next sprint.
but summary: Avoid prints.

Reply:
alright, TODO next sprint: remove prints.